examples/platform/linux/AppMain.cpp

```diff
@@ -278,8 +284,9 @@ void EventHandler(const DeviceLayer::ChipDeviceEvent * event, intptr_t arg)
     }
     else if ((event->Type == chip::DeviceLayer::DeviceEventType::kInternetConnectivityChange))
     {
-        // Restart the server on connectivity change
-        app::DnssdServer::Instance().StartServer();
+        // Restart the server on connectivity change,because recieving a netlink router event doesn't guarantee immediate network reachablility,specially ethernet,so we daly 1200 millis to start dnssd server
```

Comment:
Please fix the various spelling and grammar errors here.

Also, this comment makes it sound like we are restarting because the event doesn't guarantee immediate reachability.   But that's not what it's trying to say.  Please fix the run-on sentence to clearly communicate what's actually going on here.

Reply:
I would like to change to 
void EventHandler(const DeviceLayer::ChipDeviceEvent * event, intptr_t arg)
{
    (void) arg;
    if (event->Type == DeviceLayer::DeviceEventType::kCHIPoBLEConnectionEstablished)
    {
        ChipLogProgress(DeviceLayer, "Receive kCHIPoBLEConnectionEstablished");
    }
    else if ((event->Type == chip::DeviceLayer::DeviceEventType::kInternetConnectivityChange))
    {
        //When kInternetConnectivityChange happens,it doesn't guarantee immediate network reachablility,which restart dnssd server will fail with some low layer fail,so a delay timer to restart dnssd server  will be needed.
         DeviceLayer::SystemLayer().StartTimer(chip::System::Clock::Milliseconds32(1200),ConnectivityChnagedtimerCallback , nullptr);
       
    }
}

Reply:
Please do address "he various spelling and grammar errors".